packages/server/api/src/app/ai/mcp/tables-tools.ts

```diff
@@ -34,7 +34,10 @@ export async function getTablesTools(): Promise<MCPTool> {
   const toolSet: ToolSet = {};
   for (const [name, tool] of Object.entries(tools)) {
     if (name.includes('list')) {
-      toolSet[name] = tool;
+      toolSet[name] = {
+        ...tool,
+        toolProvider: 'tables',
```

Comment:
I think a prettier change would've been to have here the prompt name
so this way we could pass to prompt loader just the array and avoid the booleans